sdk/identity/Azure.Identity/src/ExtendedTokenRequestContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Identity
+{
+    internal readonly struct ExtendedTokenRequestContext
```

Comment:
I think this entire type is not used.

Reply:
It is not, I accidentally added it from a previous implementation of these changes.

Reply:
removed.